src/main/java/mate/academy/bookstore/model/User.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "user")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @Column(nullable = false, unique = true)
+    private String email;
+
+    @NotNull
+    @Column(nullable = false)
+    @Size(min = 8)
+    private String password;
+
+    @NotNull
+    @Column(name = "first_name", nullable = false)
```

Comment:
Don't need to specify the name, Hibernate automatically creates name with snake_case